testing/wait_for_backend_and_run_e2etests.sh

```diff
@@ -7,7 +7,7 @@ set -euxo pipefail
 is_infrastructure_up () {
```

Comment:
do we actually need this anymore? Isn't it obsolete by waiting for docker images to be healthy?

Reply:
The e2etests container needs a way to check if the other services are up before. There is currently no other function implemented that provides that check from an external view and this file is executed at the entrypoint of the e2etests dockerfile where for some reason no healthcheck exists. However, we could remove the execution of the script from the entrypoint add an always true health check and start the script from outside the container. What do you think?

Reply:
let's leave as ist.

Reply:
at least for now ;-)